experimental/packages/opentelemetry-instrumentation/src/platform/node/RequireInTheMiddleSingleton.ts

```diff
@@ -59,7 +60,7 @@ export class RequireInTheMiddleSingleton {
   }
 
   private _initialize() {
-    RequireInTheMiddle(
+    new Hook(
```

Comment:
I changed the "suggested" usage in the require-in-the-middle docs in v7.0.0 (https://github.com/elastic/require-in-the-middle/blob/main/CHANGELOG.md#v700) -- which I suppose is just a personal preference/style thing.

However, both ways work and have worked for many versions because of https://github.com/elastic/require-in-the-middle/blob/c47c46495069c39fad4e3249029d7ba2ab44ed3b/index.js#L96 -- which has been there for about 5 years (since RITM v3).